commit 7bb977c0e9fcd25bcfefca9e0e9e9c81df5e7153
Author: Geert Janssens <janssens-geert@telenet.be>
Date:   Mon Mar 21 15:10:03 2011 +0000

    [PATCH 3/4] Bug #615168: Remove configure.in check for slib
    
    Gnucash no longer requires slib.
    
    Patch by Andy Wingo.
    
    git-svn-id: svn+ssh://svn.gnucash.org/repo/gnucash/trunk@20437 57a11ea4-9604-0410-9ed3-97b8803252fd

diff --git a/configure.ac b/configure.ac
index b781c81..7bd9395 100644
--- a/configure.ac
+++ b/configure.ac
@@ -424,20 +424,6 @@ AM_GUILE_VERSION_CHECK(1.8.0, , [
 
 AS_SCRUB_INCLUDE(CFLAGS)
 
-AC_MSG_CHECKING([for SLIB support])
-if ${GUILE} -c "(use-modules (ice-9 slib)) (require 'printf)" > /dev/null 2>&1
-then
-  AC_MSG_RESULT(yes)
-else
-  AC_MSG_ERROR([
-
-Cannot find SLIB.  Are you sure you have it installed?
-See http://bugzilla.gnome.org/show_bug.cgi?id=347922
-and http://bugzilla.gnome.org/show_bug.cgi?id=483631
-
-])
-fi
-
 ### Check size of long_long - some guile's are broken.
 AC_MSG_CHECKING(if guile long_long is at least as big as gint64)
 GNC_OLDCFLAGS="$CFLAGS"
